server/command/command.go

```diff
@@ -193,38 +203,39 @@ func newCommand(args *model.CommandArgs, conf *config.Config, a splunk.Splunk) *
 
 // alertSubscriptionMessage creates message for alert subscription
 // returns message text and unique id for alert
-func alertSubscriptionMessage(siteURL string) (string, string) {
+func alertSubscriptionMessage(siteURL, secret string) (string, string) {
 	id := uuid.New()
 	post := fmt.Sprintf(
 		"Added alert\n"+
-			"You can copy following link to your splunk alert action: %s/plugins/%s%s%s?id=%s",
+			"You can copy following link to your splunk alert action: %s/plugins/%s%s%s?id=%s&secret=%s",
```

Comment:
```suggestion
			"You can copy the following link to your splunk alert action: %s/plugins/%s%s%s?id=%s&secret=%s",
```

Reply:
Is it possible for us to automate adding the splunk alert, so we don't expose show the secret in the Mattermost UI all the time?

Reply:
@mickmister  seems not possible as the message is post in channel , we can explicitly say the admin who is trying to configure  the secret but for i think it won't make any change as admin already have access to the system console where he can find secret and the message we post is user specific so it will be easy for admin to pick it directly from channel.

Reply:
> Is it possible for us to automate adding the splunk alert

@sibasankarnayak Were you answering this question when you said "seems not possible"?

> seems not possible as the message is post in channel

It's an ephemeral message so it's not too bad. Can we at least put it into a hyperlink like:

```
You can copy this [link](%s/plugins/%s%s%s?id=%s&secret=%s) to your splunk alert action.
```

Reply:
@mickmister Done with this change

Reply:
The current change is now

```
"You can copy this [here](%s/plugins/%s%s%s?id=%s&secret=%s)
```

This doesn't tell the admin what to do with this link. What would be a more descriptive instruction? Does the Splunk UI say "webhook URL" or something similar?

Reply:
@mickmister  
will it work some thing like this 

```
"You can copy this [here](%s/plugins/%s%s%s?id=%s&secret=%s) to your splunk alert action"

```

Reply:
sure